app/controllers/application_controller.rb

```diff
@@ -11,6 +11,15 @@ class ApplicationController < ActionController::Base
 
   rescue_from ActionController::InvalidAuthenticityToken, with: :invalid_auth_token
   rescue_from ActionController::UnknownFormat, with: :render_not_found
+  [
+    ActiveRecord::ConnectionTimeoutError,
+    PG::ConnectionBad, # raised when a Postgres connection times out
+    Rack::Timeout::RequestTimeoutException,
+    Redis::CannotConnectError, # on redis connection timeout
+    Redis::TimeoutError # on redis read timeout
```

Comment:
Thoughts on using the base class `Redis::BaseConnectionError` to catch all Redis connection errors? See: https://github.com/ankane/the-ultimate-guide-to-ruby-timeouts#rescuing-exceptions and http://www.rubydoc.info/github/redis/redis-rb/Redis/

Reply:
Sounds reasonable given the set of subclasses.